.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Dear @ivy-sgi, we have found the following bug: 
```

Comment:
@ivy-sgi @weissreto 

I think this opening sentence rather strange. Many customers won't know who '@ivy-sgi' is. 
And maybe the reports is not a 'we' and just a single person.

I'd omit this line --> and rather simply state 'ivy-sgi' in the 'assigness' table on top to keep Sabine informed.

Reply:
Hi @ivy-rew 

I have sent the templates about a month ago in MS Teams to get feedback about it. No reaction from the team side. So feedback is a little bit late now ...

However, @ivy-sgi should decide if she want's to address the feedback or not.

Reply:
> I have sent the templates about a month ago in MS Teams to get feedback about it. No reaction from the team side. So feedback is a little bit late now ...

Yes, sorry. I'm open to keep it as is. Just thought of better late feedback than no feedback and puzzled customers :)